chrome/extension/background.js

```diff
@@ -65,6 +65,9 @@ chrome.storage.local.get('state', (obj) => {
   const w = watch(store.getState);
 
   store.subscribe(w((newVal, oldVal) => {
+    newVal = Immutable.fromJS(newVal);
+    oldVal = Immutable.fromJS(oldVal);
```

Comment:
Appending `Immutable.fromJS` everywhere looks like the hack, could avoid it :thinking:

Reply:
We could drop immutable.js altogether, so no conversion is needed. Alternatively, we could patch store to handle that conversion implicitly. (I would try avoiding both.)